### PR TITLE
LPS-184354 [CP 2.0] Product Activation - DXP | Filter And Search Don't Working

### PR DESCRIPTION
**Pull request by samuelleolopes**
[LPS-184354](https://issues.liferay.com/browse/LPS-184354)

### CODE REVIEW

**Comment by samuelleolopes** [comment]
ci:test:sf

**Comment by samuelleolopes** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/ActivationKeysTable/index.jsx [hunk 1]
```diff
@@ -82,17 +82,18 @@ const ActivationKeysTable = ({
...
 	const [filters, setFilters] = useFilters(
 		setFilterTerm,
 		productName,
 		initialFilter
 	);
 
+	const {activationKeysByStatusPaginated, paginationConfig} = usePagination(
+		activationKeys,
+		statusFilter,
+		filters
```
Comment (kevenleone) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/ActivationKeysTable/index.jsx:94]: @samuelleolopes remove this, we don't use on usePagination

```suggestion
	
```

**Review by kevenleone — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/ActivationKeysTable/index.jsx (hunk 1 above)
Comment (samuelleolopes) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/ActivationKeysTable/index.jsx]: Fixed, thank you!

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**